apps/main/src/pages/dex.tsx

```diff
@@ -97,11 +91,6 @@ const App: NextPage = () => {
     // init locale
     const { rLocale } = getLocaleFromUrl()
     const parsedLocale = rLocale?.value ?? 'en'
-    initTranslation(i18n, parsedLocale)
```

Comment:
```tsx
const { rLocale } = getLocaleFromUrl()
const parsedLocale = rLocale?.value ?? 'en'
```
can be removed too

Reply:
let's do that separate cause then I wanna change the store too